lib/editor/components/map/LocationsLayer.js

```diff
@@ -83,7 +83,8 @@ const LocationsLayer = (props: Props) => {
         geometry_id: key,
         geometry_pt_lat: coord.lat,
         geometry_pt_lon: coord.lng,
-        id: index
+        id: index,
+        location_id: activeEntity.location_id
```

Comment:
Wait could you possibly add a comment/explain here why this is needed? Why are we doing this